Slim/Player/Protocols/HTTP.pm

```diff
@@ -78,16 +78,15 @@ sub request {
 		if ($formatClass->can('getInitialAudioBlock')) {
 			$song->initialAudioBlock($formatClass->getInitialAudioBlock($track->initial_block_fh, $track, $seekdata->{timeOffset} || 0));
 		}
+		
+		$song->initialAudioBlock('') unless defined $song->initialAudioBlock;
```

Comment:
Wouldn't this only be executed if the above condition was true? Therefore could be moved up there?

Reply:
I'm not sure. There could be many reasons why above he initialAudioBlock could still be uninitialized and I want to make sure it does not remains undef. What am I missing?